doc/support.md

```diff
@@ -82,26 +86,60 @@ sourmash v3.x supports Python 2.7 as well as Python 3.x, through Python 3.8.
+
+### Sourmash command line
+
+If you use sourmash from the command line, there are a few major changes in 4.0 that you should know about.
+
+First, **`sourmash compute` is deprecated in favor of [`sourmash sketch`](sourmash-sketch.md)**, which provides quite a bit more flexibility in creating signatures.
+
+Second, **`sourmash index` will now save databases in the Zip format (`.sbt.zip`) instead of the old JSON+subdirectory format** (see [updated docs](command-line.md#sourmash-index-build-an-sbt-index-of-signatures)). You can revert to the old behavior by explicitly specifying the `.sbt.json` filename for output when running `sourmash index`.
+
+Third, all sourmash commands that operate on signatures should now be able to directly read from lists of signatures in signature files, SBT databases, LCA databases, directories, and files containing lists of filenames (see [updated docs](command-line.md#advanced-command-line-usage)).
+
+Fourth, if you use `sourmash lca` commands, **`sourmash lca gather` has been removed**. In addition, there are some **changes in how `summarize` works**: it now uses abundances by default, and no longer combines all signatures before summarizing. Specify `--ignore-abundance` and combine your signatures using `sourmash sig merge` to recover the old behavior.
+
+Finally, **k-mer sizes have changed for amino acid sequences** in v4. if you use protein, Dayhoff, or HP signatures, we now interpret k-mer sizes differently on the command line. Briefly, k-mer sizes for protein/dayhoff/hp signatures are now the size of the k-mer in amino acid space, *not* the space of the k-mer in DNA space (as previously used). In practice this means that you need to divide all your k-mer sizes by 3 when working with k-mers in amino acid space!
+
+Note also that while `sourmash compute` still behaves the same way in v4.x as it did in sourmash 3.5.x, `sourmash sketch translate` and `sourmash sketch protein` both use the *new* approach to amino acid k-mer sizes, as do all of the the command line options for searching, manipulation, and display. Again, in practice this means that you need to divide all your k-mer sizes by 3 if they apply to amino acid k-mers.
+
+### Python API
+
+First, all k-mer sizes for `protein`, `dayhoff`, and `hp` signatures have changed in the Python layer to be "correct", i.e., to be the size of the protein k-mer. Previously they were 3\*k, i.e. based on the size of the DNA k-mer from which the protein sequence would have been created.
 
-Prior to the release of sourmash v4, we are adding deprecation
-warnings and/or future warnings to all APIs and modules in sourmash
-v3.x that are being removed in v4.0. If you are using the Python API,
-we suggest you use the following procedure to migrate:
+Second, the `MinHash` class API has changed significantly!
+* `get_mins()` has been deprecated in favor of `.hashes`, which is a dictionary that contains abundances.
```

Comment:
I simultaneously love this and hate how i'll have to fix all of my old code.

Reply:
+1000 for being more intuitive though.

Reply:
oh, well, you can totally keep using `get_mins()` until 5.0 removes it, in <checks date, makes predictions> 2025.